ecodse_funtime_alpha/train.py

```diff
@@ -16,24 +19,75 @@ def train_loop(dataset, model, optimizer):
         print(loss)
 
 
-def fit_loop(dataset, model, optimizer):
-    dataset = dataset.shuffle(12)
-    dataset = dataset.repeat(20)
-    dataset = dataset.batch(4)
+def fit_loop(dataset, model, optimizer, nepoch, batchsize):
+    nstep = len(list(dataset)) // batchsize
```

Comment:
Computing the nstep here is unnecessary as it will be done in the next PR. You can leave it to the hard coded `epochs=20, steps_per_epoch=2`

Reply:
fixed